src/test/java/uk/gov/hmcts/reform/unspec/handler/callback/ConfirmServiceCallbackHandlerTest.java

```diff
@@ -71,6 +76,146 @@ void shouldReturnNoError_whenValidServedDocumentsOther() {
         }
     }
 
+    @Nested
+    class SecondMidEventCallback {
+
+        private final LocalDate claimIssueDate = LocalDate.of(2000, 6, 22);
+
+        @Nested
+        class ServiceDate {
+
+            private final LocalDate today = LocalDate.now();
+            private final LocalDate futureDate = today.plusYears(1);
+
+            @Test
+            void shouldReturnNoErrors_WhenServiceDateInPastAndAfterIssueDate() {
```

Comment:
when with small `w` please

```suggestion
            void shouldReturnNoErrors_whenServiceDateInPastAndAfterIssueDate() {
```